source/_components/light.rest.markdown

```diff
@@ -0,0 +1,120 @@
+    required: false
+    type: string
+  color_temp_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the color_temp of the light from the value returned by the GET request on the resource URL. This template should compute to an integer. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.color_temp}}`{% endraw %}.'
+    required: false
+    type: string
+  effect_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the effect of the light from the value returned by the GET request on the resource URL. This template should compute to an string. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.effect}}`{% endraw %}.'
+    required: false
+    type: string
+  rgb_color_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the rgb_color of the light from the value returned by the GET request on the resource URL. This template should compute to an array of three integers, representing red, green and blue. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.rgb_color}}`{% endraw %}.'
+    required: false
+    type: string
+  name_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the name of the light from the value returned by the GET request on the resource URL. This template should compute to a string. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.name}}`{% endraw %}.'
+    required: false
+    type: string
+  supported_features_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the supported features of the light from the value returned by the GET request on the resource URL. This template should compute to an array of strings. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.supported_features}}`{% endraw %}.'
+    required: false
+    type: string
+  effect_list_template:
+    description: 'A [template](/docs/configuration/templating/#processing-incoming-data) that determines the effect list of the light from the value returned by the GET request on the resource URL. This template should compute to an array of strings. If the value is valid JSON, it will be available in the template as the variable `value_json`. Default is {% raw %}`{{value_json.effect_list}}`{% endraw %}.'
+    required: false
+    type: string
+  supported_features:
```

Comment:
I think you should mention here that this is overwritten with `supported_features_template`

Reply:
Thanks @tinloaf, I've now updated this as per your recommendations (commit `0dbe10c`).